roles/katello_provisioning/tasks/main.yml

```
@@ -117,16 +117,16 @@
     {{ katello_provisioning_hammer }} repository create
     --product="CentOS"
     --content-type="yum"
-    --name "EPEL 7"
-    --url "https://dl.fedoraproject.org/pub/epel/7/x86_64"
+    --name "EPEL 9"
+    --url "https://dl.fedoraproject.org/pub/epel/9/x86_64"
```

Looks like this URL doesn't work but this does:
```suggestion
    --url "https://dl.fedoraproject.org/pub/epel/9/Everything/x86_64"
```